recipes/arraylake/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "arraylake" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arraylake-{{ version }}.tar.gz
+  sha256: 34a00aca0c3bda56c5fb94049a94decf0fe64aa8ff6439e2440fa5e1d8c39237
```

Comment:
```suggestion
  sha256: 663efad630aa04ebe47d5cf6903494b26e81177164e223cf51059678ff7561f3
```

Reply:
<img width="963" alt="image" src="https://github.com/conda-forge/staged-recipes/assets/2448579/7c79ead0-fb05-49ce-8507-e0dd9c74b5e9">
?

Reply:
ah we need the tarball sha